package.json

```diff
@@ -19,22 +19,21 @@
     "standalone"
   ],
   "scripts": {
-    "test": "xo && ava",
+    "test": "jest",
```

Comment:
Are you going to replace `xo` with `eslint`?

Reply:
Yes!